app/models/item.rb

```diff
@@ -1,6 +1,8 @@
 class Item < ActiveRecord::Base
     belongs_to :user
     validates_presence_of :name
+    validates :type, inclusion: { in: %w(regular clothing ticketing),
```

Comment:
While having a "regular" type makes sense from a development perspective. (Because then we can reasonably deal with items that are neither clothing nor tickets.) It might make more sense to just describe this as "miscellaneous"